vmdb/app/models/vm_microsoft.rb

```diff
@@ -7,4 +7,8 @@ def self.calculate_power_state(raw_power_state)
     else                        super
     end
   end
+
+  def supports_migrate?
+    false
+  end
```

Comment:
That didn't answer my question.  Right now, with true in the base class, TemplateMicrosoft returns "true".  Is that correct?